src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private String color;
+
+    abstract double getArea();
```

Comment:
Please be consistent, if you want to define behavior (methods) in separate interfaces like you did in case of method draw and Drawable interface you should create an interface AreaCalculator with method getArea, or avoid approach to use interfaces and declare everything in abstract class. First approach is more flexible and better from my point of view